src/NzbDrone.Core/SeriesStats/SeasonStatistics.cs

```diff
@@ -54,5 +58,20 @@ public class SeasonStatistics : ResultSet
                 return previousAiring;
             }
         }
+
+        public List<string> ReleaseGroups
+        {
+            get
+            {
+                List<string> releaseGroups = null;
```

Comment:
a list should never be null, `var releasegroups = new List<string>()`;

Or convert to a ternary instead:

```
return ReleaseGroupsString.IsNullOrWhiteSpace() ? new List<string>() : ReleaseGroupsString.Split('|').Distinct().ToList()`
```

(also add the sort here).